build_scripts/release_scripts/Jenkinsfile

```diff
@@ -36,9 +36,10 @@ pipeline {
             steps {
```

Comment:
Added a --no-cache flag.

Regarding ubuntu22 - we're not deploying RonDB 21.04.9 for ARM? Since we agreed to just do that for 22.10..? But this file will be the basis of that. This file is for the case where Mikael / me want to build something for personal usage.

@mikaelronstrom, had I understood that correctly?